### PR TITLE
Feat: Transfer History List - 478

### PR DESCRIPTION
**Pull request by AlexZorkin**
- migration adding user_profile to transfer_history table
- hooked up real data for transfer history list
- created a separate transfer history component
- added dynamic labelling based on transfer status for history list

### CODE REVIEW

File: backend/lcfs/web/api/transfer/repo.py [hunk 1]
```diff
@@ -143,7 +147,8 @@ async def add_transfer_history(self, transfer_id: int, transfer_status_id: int)
         """
         new_history_record = TransferHistory(
             transfer_id=transfer_id,
-            transfer_status_id=transfer_status_id
+            transfer_status_id=transfer_status_id,
+            user_profile=self.request.user
```
Comment (kevin-hashimoto) [inline backend/lcfs/web/api/transfer/repo.py:151]: seems to be an error when saving a record.
`'TransferRepository' object has no attribute 'request'`

probably just adding the `request` object to the init up top

File: backend/lcfs/web/api/transfer/repo.py
Comment (kevin-hashimoto) [inline backend/lcfs/web/api/transfer/repo.py:1]: `update_transfer` and `create_transfer` will need `transfer_history` in the `self.sb.refresh` list or else a `MissingGreenlet` error occurs and throws an error on the front end. Tranfer will save in the background regardless though.

**Review by kevin-hashimoto — CHANGES_REQUESTED** [review]

File: backend/lcfs/web/api/transfer/repo.py (hunk 1 above)
Comment (AlexZorkin) [reply backend/lcfs/web/api/transfer/repo.py:151]: I updated the service to pass down the user_profile_id. We probably should avoid having the request injected into the repo layer, seems a bit anti-repository pattern.

File: backend/lcfs/web/api/transfer/repo.py
Comment (AlexZorkin) [reply backend/lcfs/web/api/transfer/repo.py:1]: Thanks for this catch. I have updated the refresh lists.

**Review by kevin-hashimoto — APPROVED** [review]
LGTM!

**Pull request merged**